dramatiq/brokers/rabbitmq.py

```diff
@@ -216,17 +218,19 @@ def consume(self, queue_name, prefetch=1, timeout=5000):
         Returns:
           Consumer: A consumer that retrieves messages from RabbitMQ.
         """
-        self.declare_queue(queue_name, ensure=True)
+        self.declare_queue(queue_name, ensure=True, strict=True)
         return self.consumer_class(self.parameters, queue_name, prefetch, timeout)
 
-    def declare_queue(self, queue_name, *, ensure=False):
+    def declare_queue(self, queue_name, *, ensure=False, strict=False):
```

Comment:
Overall, this looks like a fine change, but I'm wondering if, instead of having `ensure` and `strict` be separate flags, we shouldn't just extend `ensure` to `bool | "strict"`.

Reply:
corrected